charts/prometheus-node-exporter/templates/daemonset.yaml

```
@@ -146,13 +146,21 @@ spec:
             {{- end }}
 {{- if .Values.sidecars }}
 {{ toYaml .Values.sidecars | indent 8 }}
-          {{- if .Values.sidecarVolumeMount }}
+          {{- if or (.Values.sidecarVolumeMount)(.Values.sidecarHostVolumeMounts)}}
```

@huangkevin404 
Could you explain in what scenario we need extra bracket `()` for each variable in order to do `or` operation? Why we could not just do this:

```suggestion
          {{- if or .Values.sidecarVolumeMount .Values.sidecarHostVolumeMounts }}
```

---

@zanhsieh 
I'm not super familiar with helm syntax. It seems that you're right, the extra brackets are not needed here, thx for the suggestion !